pkg/api/v1/types.go

```diff
@@ -361,6 +381,10 @@ type VirtualMachineInstanceMigrationState struct {
 	Completed bool `json:"completed,omitempty"`
 	// Indicates that the migration failed
 	Failed bool `json:"failed,omitempty"`
+	// Indicates that the migration has been requested to abort
+	AbortRequested bool `json:"abortRequested,omitempty"`
```

Comment:
Isn't that covered by DeletionTimestamp != nil and also indicated by the condition?

Reply:
Yes, but this is being used in the VMI, where we don't have the migration object

Reply:
oh, the code-folding made me misunderstand it. All good then.